packages/blockchain-wallet-v4-frontend/src/scenes/Signup/components/Header/index.tsx

```diff
@@ -36,20 +38,31 @@ const SubHeader = styled(Text)`
   `}
 `
 
-const Header = () => (
+const Header = (props: SubviewProps) => (
   <Wrapper>
     <Column>
       <HeaderText color='whiteFade900' weight={700} data-e2e='signupSecureHeader'>
-        <FormattedMessage
-          id='scenes.register.securelybuy'
-          defaultMessage='Securely Buy, Sell, and Store Crypto.'
-        />
+        {!props.isLatam && (
+          <FormattedMessage
+            id='scenes.register.securelybuy'
+            defaultMessage='Securely Buy, Sell, and Store Crypto.'
+          />
+        )}
+        {props.isLatam && (
+          `Envía, recibe y obtén hasta un 14g% de ganancias con tus criptos.`
+        )}
       </HeaderText>
+      
       <SubHeader color='whiteFade800' weight={500} data-e2e='signupSecureSubHeader'>
+      {!props.isLatam && (
         <FormattedMessage
           id='scenes.register.getstarted'
           defaultMessage='Get Started by Signing Up Now.'
         />
+      )}
+      {props.isLatam && (
+        `Crea tu cuenta y verifica tu perfil para comenzar a operar.`
```

Comment:
this needs translation

Reply:
I am waiting to have access to lokalize app.